docs/csharp/toc.yml

```diff
@@ -1052,9 +1052,6 @@ items:
         href: language-reference/keywords/default.md
     - name: Contextual Keywords
       items:
-      - name: Quick reference
-        displayName: contextual keywords
-        href: language-reference/keywords/contextual-keywords.md
```

Comment:
You could, if you wanted, keep this link and point to the index.md#contextual-keywords